packages/react-scripts/template/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We can propose this change as part of enhancing contributor workflow, I could see some potential value. 😄